### ISSUE TITLE
[bug] conflicted report for `line-too-long` and `useless-suppression`

### ISSUE THREAD

**Issue by msyyc** [pylint-dev/pylint#10230]
When I run command `pylint .\shiyan.py --max-line-length 120 --enable  useless-suppression --disable too-few-public-methods,missing-module-docstring` for the following python code
```python
class Test:
    """class"""

    a = "11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111"  # type: ignore # pylint: disable=line-too-long
    b = "11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111"  # type: ignore

    def __init__(self, a: str, b: str) -> None:
        """ module """
        self.a = a
        self.b = b


print(Test.a)

```
The following error is reporetd:

![Image](https://github.com/user-attachments/assets/d8167e7c-6abc-4eca-aa16-cf407a4354c2)

If no `# pylint: disable=lint-too-long` added, pylint reports `line-too-long` error; if added, pylint reports `useless-suppression`.

## Expected behavior
I think pylint shall not report useless-suppression if we add `pylint: disable=lint-too-long`.

## More info about environments:

![Image](https://github.com/user-attachments/assets/ad59817f-770a-4deb-a4dd-fd7c99162b1a)
